packages/orchestrator-ui-components/src/components/WfoTableCodeBlock/styles.ts

```diff
@@ -0,0 +1,13 @@
+import { css } from '@emotion/react';
+
+export const getStyles = () => {
+    const tableCodeBlockMarginStyle = css({
+        margin: 15,
+        marginTop: 20,
+        marginRight: 20,
```

Comment:
I think it might be better to use values from the theme here, else introduce constants. `discussion`

Reply:
Done